src/member.spec.ts

```diff
@@ -261,4 +261,166 @@ describe('member', () => {
       expect(error).toHaveProperty('name', 'MongoNotConnectedError');
     });
   });
+
+  describe('updateMember', () => {
+    const testEmail = 'different@email.com';
+
+    test('Valid should be updated', async () => {
```

Comment:
This is testing the scenario where all the fields are set, but I think the opposite (nothing is changed) is also useful.